### PR TITLE
LF6 update

### PR DESCRIPTION
**Pull request by AbbyANoble**
Answer choices A and C were the same (6c + 4a vs 4a + 6c). The answer said A, so I assume C was supposed to be different and switched the variables to make what I think was the intended wrong answer. Tagging @kathypinzon as a reviewer since she was the original author!

Link to original: [Activity 3.6.2(b)](https://library.tbil.org/2024/precalculus/LF6.html#LF4-carnival-prices-3) 

Screenshot of original: 
image

### CODE REVIEW

File: source/precalculus/source/03-LF/06.ptx [hunk 1]
```diff
@@ -55,7 +55,7 @@
           <li><p> <m>6c+4a=208</m> </p></li>
           <li><p> <m>a+c=208
           </m> </p></li>
-          <li><p> <m>4a+6c=208</m> </p></li>
+          <li><p> <m>4c+6a=208</m> </p></li>
```
Comment (StevenClontz) [inline source/precalculus/source/03-LF/06.ptx:58]: Would it be good (throughout this activity) to always list the `a` term first?

Comment (StevenClontz) [reply source/precalculus/source/03-LF/06.ptx:58]: Or vice/versa (children then adults), but either way, write things consistently to reduce cognitive overload of learners?

Comment (AbbyANoble) [reply source/precalculus/source/03-LF/06.ptx:58]: Consistency in order sounds good to me.

**Review by StevenClontz — CHANGES_REQUESTED** [review]
As discussed, let's edit so variables appear in a consistent order.

File: source/precalculus/source/03-LF/06.ptx (hunk 1 above)
Comment (AbbyANoble) [reply source/precalculus/source/03-LF/06.ptx:58]: About to make an update.

File: source/precalculus/source/03-LF/06.ptx [hunk 2]
```diff
@@ -32,11 +32,11 @@
...
       <p>
         <ol marker="A." cols="1">
-          <li><p> <m>4a+2c=128.50</m> </p></li>
-          <li><p> <m>a+c=128.50
+          <li><p> <m>2c+4a=128.50</m> </p></li>
+          <li><p> <m>c+a=128.50
           </m> </p></li>
           <li><p> <m>4c+2a=128.50</m> </p></li>
-          <li><p> <m>a+c=336.50</m> </p></li>
+          <li><p> <m>c+a=336.50</m> </p></li>
```
Comment (StevenClontz) [inline source/precalculus/source/03-LF/06.ptx:39]: Should we add $ signs?

File: source/precalculus/source/03-LF/06.ptx [hunk 3]
```diff
@@ -53,10 +53,10 @@
       <p>
         <ol marker="A." cols="1">
           <li><p> <m>6c+4a=208</m> </p></li>
-          <li><p> <m>a+c=208
+          <li><p> <m>c+a=208
           </m> </p></li>
-          <li><p> <m>4a+6c=208</m> </p></li>
-          <li><p> <m>a+c=336.50</m> </p></li>
+          <li><p> <m>4c+6a=208</m> </p></li>
+          <li><p> <m>c+a=336.50</m> </p></li>
```
Comment (StevenClontz) [inline source/precalculus/source/03-LF/06.ptx:59]: I think we want $ here too. (use `\$` in math mode)

**Comment by StevenClontz** [comment]
Thanks @AbbyANoble, use the re-request review button when this is ready:

image

**Comment by AbbyANoble** [comment]
> Thanks @AbbyANoble, use the re-request review button when this is ready:

Oops. I re-requested without reading all the way. 

I can go either way on the dollar signs. Personally, I feel like they junk up a system of equations.

**Comment by StevenClontz** [comment]
In that case, do we need to remove them further down? [Activity 3.6.2(c)](https://f32358f8.tbil.pages.dev/preview/precalculus/instructor/LF6#LF4-carnival-prices-4)

**Comment by AbbyANoble** [comment]
> In that case, do we need to remove them further down? [Activity 3.6.2(c)](https://f32358f8.tbil.pages.dev/preview/precalculus/instructor/LF6#LF4-carnival-prices-4)

Those don't bother me because they are an answer (where I would use units) instead of an abstraction of the problem. But I'm good with putting them everywhere. I'll throw them in.

**Review by StevenClontz — APPROVED** [review]
Maybe I'm a pedant, but I definitely prefer consistent use of units; I know our colleagues that apply mathematics to their fields appreciate it when we take care with dimensional analysis.

**Pull request merged**